src/main/java/org/gradle/playframework/tools/internal/routes/DefaultRoutesPostProcessor.java

```diff
@@ -0,0 +1,50 @@
+package org.gradle.playframework.tools.internal.routes;
+
+import org.gradle.util.RelativePathUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+class DefaultRoutesPostProcessor implements Serializable {
```

Comment:
Should we add a javadoc to clarify what exactly this class does and what it is processing?